server/repository/src/db_diesel/assets/asset_catalogue_item.rs

```diff
@@ -31,7 +31,7 @@ pub struct AssetCatalogueItemFilter {
     pub code: Option<StringFilter>,
     pub manufacturer: Option<StringFilter>,
     pub model: Option<StringFilter>,
-    pub r#type: Option<StringFilter>,
+    pub r#type: Option<EqualFilter<String>>,
```

Comment:
I think we should put this back too...
```suggestion
    pub r#type: Option<StringFilter>,
```